app/code/Magento/Wishlist/Helper/Data.php

```diff
@@ -134,7 +141,8 @@ public function __construct(
         \Magento\Customer\Helper\View $customerViewHelper,
         WishlistProviderInterface $wishlistProvider,
         \Magento\Catalog\Api\ProductRepositoryInterface $productRepository,
-        Escaper $escaper = null
+        Escaper $escaper = null,
+        StockItemRepository $stockItemRepository
```

Comment:
Remove backward compatible Object loader for escaper, introduce such for your new dependency.


```suggestion
        Escaper $escaper,
        StockItemRepository $stockItemRepository = null
```